### PR TITLE
Raihan error fixing

### PR DESCRIPTION
**Pull request by DzulhiRaihan**
# **Name of PR**

## **Description**

*

---

### **Additional context**

*

### CODE REVIEW

File: src/pages/_app.tsx [hunk 1]
```diff
@@ -41,7 +41,7 @@ export default function App({ Component, pageProps }: AppProps) {
   return (
     <Providers>
       <main>
-        <Navbar home={home} transparent={temp} about={about} login={true}></Navbar>
+        {/* <Navbar home={home} transparent={temp} about={about} login={true}></Navbar> */}
```
Comment (VosMovok) [inline src/pages/_app.tsx:44]: biar apa disembunyiin terus 😭

File: src/components/formLisy.tsx
Comment (VosMovok) [inline src/components/formLisy.tsx:1]: Hapus ni conflict ntar sm sebelah

File: src/components/maps.tsx
Comment (VosMovok) [inline src/components/maps.tsx:1]: knp km hapus ni??

File: src/pages/aboutUs.tsx
Comment (VosMovok) [inline src/pages/aboutUs.tsx:1]: lahh dihapus jg

File: src/pages/product.tsx [hunk 2]
```diff
@@ -1,8 +1,9 @@
 import Head from "next/head";
 import Link from "next/link";
+import Image from "next/image";
 import Footer from "../components/footer";
 import { useState, useRef, useEffect } from "react";
-import {Map} from "@/components/maps";
```
Comment (VosMovok) [inline src/pages/product.tsx:5]: error kan ni

File: src/pages/aboutUs.tsx
Comment (DzulhiRaihan) [reply src/pages/aboutUs.tsx:1]: Iya hapus

File: src/pages/product.tsx (hunk 2 above)
Comment (DzulhiRaihan) [reply src/pages/product.tsx:5]: Udah di hapus

**Comment by VosMovok** [comment]
Conflict and deleted important files

**Pull request closed without merging**